src/sentry/api/endpoints/project_filter_details.py

```diff
@@ -26,6 +27,36 @@ def put(self, request, project, filter_id):
         if not serializer.is_valid():
             return Response(serializer.errors, status=400)
 
-        filter.enable(serializer.object)
+        if not serializer.is_valid():
+            return Response(serializer.errors, status=400)
+
+        current_state = filter.is_enabled()
+        new_state = filter.enable(serializer.object)
+        audit_log_state = AuditLogEntryEvent.PROJECT_ENABLE
+
+        if filter.id == 'legacy-browsers':
+            added = new_state - current_state
+            removed = current_state - new_state
+
+            if removed:
+                returned_state = removed
+                audit_log_state = AuditLogEntryEvent.PROJECT_DISABLE
+            elif added:
+                returned_state = added
+
+        if filter.id == 'browser-extensions' or filter.id == 'localhost' or filter.id == 'web-crawlers':
```

Comment:
```suggestion
        if filter.id in ('browser-extensions', 'localhost', 'web-crawlers'):
```

Using `in` will save you some words.